api/specs/web-server/_projects_crud.py

```diff
@@ -39,19 +39,14 @@
 )
 
 
-#
-# API entrypoints
-#
-
-
 @router.post(
     "/projects",
     response_model=Envelope[TaskGet],
     summary="Creates a new project or copies an existing one",
     status_code=status.HTTP_201_CREATED,
 )
 async def create_project(
-    _params: Annotated[_ProjectCreateParams, Depends()],
+    _params: Annotated[ProjectCreateParams, Depends()],
```

Comment:
Q: you add it via depends so you don't have to specify each individual parameter one by one and this is kept into sync? Did I get it correct?

Reply:
yes